src/mscorlib/src/System/Globalization/SortVersion.cs

```diff
@@ -0,0 +1,102 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Globalization 
+{
+    using System;
+    using System.Diagnostics.Contracts;
+
+    [Serializable]
```

Comment:
Yes I am planning to work on Linux later so I can move stuff around at then. I am currently focusing to get Windows implementation fully working but I expect we'll share some code between Windows and Linux later.